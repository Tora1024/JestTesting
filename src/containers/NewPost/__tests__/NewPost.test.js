import React from 'react';
import { shallow } from 'enzyme';
import ComponentUnderTest from '../NewPost';

describe('containers/NewPost/NewPost.js', () => {
  let wrapper;

  describe('in general', () => {
    beforeEach(() => {
      wrapper = shallow(<ComponentUnderTest />);
    });
    it('should render correct snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
