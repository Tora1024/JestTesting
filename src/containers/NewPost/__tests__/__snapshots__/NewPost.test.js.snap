// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/NewPost/NewPost.js in general should render correct snapshot 1`] = `
<Connect(Form(Connect(NewPost)))
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="PostsNewForm"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldValidate={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
  validate={[Function]}
/>
`;
